include/exadg/aero_acoustic/single_field_solvers/acoustics.h

```diff
@@ -68,6 +68,48 @@ class SolverAcoustic
     time_integrator->setup(application->get_parameters().restarted_simulation);
   }
 
+  void
+  advance_multiple_timesteps(double const macro_dt)
+  {
+    // in case the macro time step is smaller than the acoustic timestep, simply run
+    // with macro time step, otherwise, ensure that n_sub_time_steps * sub_dt = macro_dt
+    double sub_dt = time_integrator->get_time_step_size();
+    if(macro_dt > sub_dt)
+      sub_dt = adjust_time_step_to_hit_end_time(0.0, macro_dt, sub_dt);
```

Comment:
Do we need `else{ sub_dt = macro_dt;}` to make sure that fluid and acoustic stay in sync regarding macro time steps?

Reply:
Yes, this makes it more clear: [fix sumtimestep computation](https://github.com/exadg/exadg/pull/645/commits/e8d1525c2ec742cb5520c89ea43c19fcc5c9ea31) :)